wcvendors/wcvendors
# Updated: Changed css class for vendor list

### All Submissions:


### Changes proposed in this Pull Request:

Updates the CSS class to inherit the correct styling from the WooCommerce styles. 

Closes #867.

### How to test the changes in this Pull Request:

1. Load vendor list page
2. See pagination is now styled. 


### Other information:
